src/gurt/fault_inject.c

```diff
@@ -45,14 +45,71 @@
 
 #define FI_MAX_FAULT_ID 8192
 
+/* (1 << D_FA_TABLE_BITS) is the number of buckets of fa hash table */
+#define D_FA_TABLE_BITS		(13)
+
 #include <gurt/common.h>
+#include <gurt/hash.h>
+
+struct d_fault_attr {
```

Comment:
Recently, I created gurt/types.h.  Types should be there.

Also, we might consider creating an fi or fault_inject .h file rather than continuing to overload common.h